packages/console/src/components/Button/index.tsx

```diff
@@ -0,0 +1,28 @@
+import { I18nKey } from '@logto/phrases';
+import classNames from 'classnames';
+import React, { HTMLProps } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import * as styles from './index.module.scss';
+
+type Props = Omit<HTMLProps<HTMLButtonElement>, 'type' | 'size' | 'title'> & {
+  title: I18nKey;
+  type?: 'primary' | 'danger';
+  size?: 'small' | 'medium' | 'large';
+};
+
+const Button = ({ type = 'primary', size = 'medium', title, ...rest }: Props) => {
```

Comment:
do we need to define "rest", to only allow pre-defined props like styles, classNames ...

Reply:
the type of `rest` should be good, please see the definition above; also it won't work if there's extra prop since we always spread it on `<button>` (see below)